src/common/paths/dbrps_dbrpID.yml

```diff
@@ -23,7 +27,7 @@ get:
       content:
         application/json:
           schema:
-            $ref: "../schemas/DBRP.yml"
+            $ref: "../schemas/DBRPGet.yml"
```

Comment:
`DBRPGet` is the response schema for the `get` method (and the `patch` method, see below) - which basically wraps the `DBRP` object in an object with the key `content`. This matches what the server produces.